lte/gateway/docker/docker-compose.yaml

```diff
@@ -25,6 +25,9 @@ x-generic-service: &service
   logging: *logging_anchor
   restart: always
   network_mode: host
+  healthcheck:
+    timeout: "4s"
+    retries: 3
```

Comment:
I'm not sure if we should deduplicate this. Currently deduplicating is possible because the parameters are not fine-tuned to the individual checks. Once we start to fine-tune the timeout (like further down in this PR with sctpd) then we actually have two places where we define the timeout: One default and one overwrite. Defaults + overwrites are more complex than just having one definite place to look. Also, by deduplicating, we now split the `healthcheck` blocks into two: This part that defines the timeouts and retries, and the part in the respective services that defines the tests. Without de-duplicating you can easily see the whole healthcheck definition for a service at a glance.

Reply:
- The classic discussion on `code abstraction vs duplication`. I am sure there are heaps of internet resources arguing for both sides.
- `Once we start`: But we do not at the moment. Would it not be sensible to focus on the current state, on the here and now, instead of anticipating what we might or might not want to do in future?

Reply:
> But we do not at the moment
Yes we do, in this very PR, with sctpd. That's what I wrote in paranthesis.

Reply:
For me it works if we set all timeouts to 4s, so maybe there is no need for fine-tuning at this point.